main/kotlin/com/example/people_managment_system/controller/SystemController.kt

```diff
@@ -0,0 +1,52 @@
+package com.example.people_managment_system.controller

```

Comment:
I recommend that whenever you have an API type of class such as in your case is HTTP. Then create a package structure such as: `.../api/http/v1/<Controller Classes>`.

When we create API, we version them because in the future we will look to up their versions with new improvements while supporting older versions for legacy up until everyone has upgraded to the latest version. This is a standard practice to follow because once you release an API, you should not look change the API contract, what I mean by that is changing the API URL, because this would break for the customer using it.